vm/src/builtins/bytes.rs

```diff
@@ -606,6 +607,18 @@ impl AsSequence for PyBytes {
     };
 }
 
+impl AsNumber for PyBytes {
+    const AS_NUMBER: PyNumberMethods = PyNumberMethods {
+        remainder: Some(|number, other, vm| {
+            Ok(Self::number_downcast(number)
+                .inner
+                .cformat(other.to_pyobject(vm), vm)?
```

Comment:
```suggestion
                .mod_(other, vm)?
```
reusing `mod_` will be better